flink-connector-mysql-cdc/src/test/java/com/ververica/cdc/connectors/mysql/MySqlValidatorTest.java

```diff
@@ -178,12 +183,21 @@ private void startSource(UniqueDatabase database) throws Exception {
                             .deserializer(new MySqlTestUtils.ForwardDeserializeSchema())
                             .serverTimeZone("UTC")
                             .build();
-
-            mySqlSource.createEnumerator(new MockSplitEnumeratorContext<>(1)).start();
+            SplitEnumerator<MySqlSplit, PendingSplitsState> enumerator =
+                    mySqlSource.createEnumerator(new MockSplitEnumeratorContext<>(1));
+            try {
+                enumerator.start();
+            } finally {
+                enumerator.close();
+            }
```

Comment:
SplitEnumerator is AutoClosable

```suggestion
            try (SplitEnumerator<MySqlSplit, PendingSplitsState> enumerator =
                    mySqlSource.createEnumerator(new MockSplitEnumeratorContext<>(1))) {
                enumerator.start();
            }
```